packages/types/src/client.ts

```diff
@@ -179,6 +179,13 @@ export interface Client<O extends ClientOptions = ClientOptions> {
    */
   recordDroppedEvent(reason: EventDropReason, dataCategory: DataCategory, event?: Event): void;
 
+  /**
+   * Captures serialized metrics and sends them to Sentry.
+   *
+   * @experimental This API is experimental and might experience breaking changes
+   */
+  captureSerializedMetrics?(serializedMetrics: string): void;
```

Comment:
l/m: It honestly feels kinda weird that we pass serialized data in this client method. For all the other event types we pass unserialized data 🤔 Should we rethink this?

Reply:
Couldn't think of a better way to do this without adding to bundle size. I'm all ears if you have an alternative.

Reply:
I don't know how it would affect bundle size but we could maybe turn this into `captureMetrics(metrics: AggregatedMetrics)`. This would just be to be a bit more "conventional" though. User benefit is probably slim to none. Maybe this would tie a bit into my point about having an Aggregator only do aggregation and no creation of metrics...